data_steward/cdr_cleaner/manual_cleaning_rules/remove_operational_pii_fields.py

```diff
@@ -74,7 +74,7 @@ def load_operational_pii_fields_lookup_table(project_id, sandbox_dataset_id):
                                  fields=None)
 
 
-def parse_intermediary_table_query(dataset_id, project_id, sandbox_dataset_id):
+def get_intermediary_table_query(dataset_id, project_id, sandbox_dataset_id):
```

Comment:
Now that these are not called from tests, should they be made private?

Reply:
yeah i can make them private.